### DISCUSSION TITLE
CSGO Server on google cloud

### DISCUSSION THREAD

**Discussion by SiVy1** [GameServerManagers/LinuxGSM#3194] [Q&A] [answered]
Hello, I recently started setting up my first csgo server on google cloud, unfortunately the server does not show up in the server list, nor can I connect to it using the connect (ip) command. I have already installed the server on 2 machines and both have the same problem. I think my firewall is well set, and I will give you a link to what is displayed when you type ./csgoserver details.
![image](https://user-images.githubusercontent.com/77787836/105355659-ba417f80-5bf2-11eb-8485-86495d36f746.png)

and ports 
port="27015"
clientport="27005"
sourcetvport="27020"

**Comment by H4D3CK** [comment]
what did you put in source IP ranges?

**Comment by johnoclockdk** [comment, marked as answer]
this have be solve in discord
